mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -145,7 +145,7 @@ CLASS net/minecraft/unmapped/C_xmldumst net/minecraft/world/GameRules
 			ARG 1 key
 			ARG 2 type
 	CLASS C_bctfwntr Acceptor
-		METHOD call call (Lnet/minecraft/unmapped/C_xmldumst$C_avotqrag;Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
+		METHOD call accept (Lnet/minecraft/unmapped/C_xmldumst$C_avotqrag;Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;Lnet/minecraft/unmapped/C_xmldumst$C_mymgluou;)V
```

Comment:
All methods that call this are named `accept`, and it is the only thing in the class called `Acceptor`.